src/test/regress/pg_regress_multi.pl

```diff
@@ -50,6 +50,7 @@ ()
     print "  --connection-timeout	Timeout for connecting to worker nodes\n";
     print "  --mitmproxy        	Start a mitmproxy for one of the workers\n";
     print "  --worker-count         Number of workers in Citus cluster (default: 2)\n";
+	print "  --seclabel-test        This test runs seclabel propagation tests";
```

Comment:
I don't think we really need this flag. If possible, let's just add `citus.running_under_citus_test_suite=true` always.

Reply:
Alright, let me see if that is possible.